presto-docs/src/main/sphinx/release/release-0.179-t.rst

```diff
@@ -4,13 +4,139 @@ Release 0.179-t
+* Determine join distribution type based on statistics when ``join_distribution_type`` is set to ``AUTOMATIC``.
+* Statistics calculated by Presto for each stage of the query plan can be seen in ``EXPLAIN`` results.
 
-There is initial support for large joins by spilling partial results to disk. This can be enabled
-with the ``beta.spill-enabled`` configuration flag.
+Spill to Disk
+--------------
+* Improve spill-to-disk support for aggregations.
+* Support spilling to disk for inner joins that exceed available memory. This can be enabled with the ``beta.spill-enabled`` configuration flag.
+* Verify that spilled data is not corrupted while un-spilling. 
+* Remove the ``experimental.operator-memory-limit-before-spill`` config property and the ``operator_memory_limit_before_spill`` session property.
+* Allow configuring the amount of memory that can be used for merging
+  spilled aggregation data from disk using the ``experimental.aggregation-operator-unspill-memory-limit`` config property
+  or the ``aggregation_operator_unspill_memory_limit`` session property.
+* Change spilling for aggregations to only occur when the cluster runs out of memory.
+* Add spilled data size to ``EXPLAIN ANALYZE``.
+* Add spilled data size to the Presto CLI when run with the ``--debug`` flag.
+* Add spilled data size to the Presto WebUI on the Query Details and Live Plan pages.
 
-**Fix execution of several window functions on array and map types**
+General Changes
+----------------
+* Support prepared statements that are longer than 4K bytes.
+* Improve the performance of joins with only non-equality conditions by using
+  a nested loops join instead of a hash join.
+* Add ``VERBOSE`` option for ``EXPLAIN ANALYZE`` that provides additional low-level details about query performance.
+* Add per-task distribution information to the output of ``EXPLAIN ANALYZE``.
+* Enable more join predicates to be pushed down to the source tables.
+* Add :link: TPCDS Connector. This connector provides a set of schemas to support the TPC Benchmark™ DS (TPC-DS). It exposes table and column statistics for schemas ``tiny`` and ``sf1``.
+* Avoid potentially expensive computation on coordinator by offloading certain plan fragments to worker nodes.
+* Make ``DECIMAL`` the default literal for non-integral numbers.
+* Add support for sorting data on all workers of the cluster, instead of just one node. This feature is disabled by default and can be enabled with the config property ``experimental.distributed-sort`` and the session property ``distributed_sort``. 
+* Improve performance for selective filters by merging pages. This feature can be disabled by setting the config property ``experimental.filter-and-project-min-output-page-row-count`` or the session property ``filter_and_project_min_output_page_row_count`` to 0.
```

Comment:
Improve performance for selective filters by merging pages with few output rows.  

also, say at the end "The default value is 256".